ipinfo/cmd_match_ip.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/ipinfo/cli/lib"
+	"github.com/spf13/pflag"
+)
+
+func printHelpMatchIP() {
+	fmt.Printf(
+		`Usage: %s matchip --filter <file(s) | stdin> --criteria <file(s) | stdin>
```

Comment:
A more ideal form is one that matches `grep`'s solution to this:

```bash
# single expression + single file
matchip 127.0.0.1 file1.txt

# single expression + multiple files
matchip 127.0.0.1 file1.txt file2.txt file3.txt

# multi expression + any files
matchip -e 127.0.0.1 -e 8.8.8.8 file1.txt file2.txt file3.txt
```

I.e., use an 'expression' or 'input' flag to label each input, equivalent to the `--filter` flag you currently have. However `-f` wouldn't be a great option since it commonly stands for `file` so could cause confusion. `-e` / `--expression` is what `grep` uses, so in the spirit of trying to re-use people's habits / knowledge of `grep` like we did in `grepip`, maybe that's ideal.